platform/core/src/services/DisplaySetService/DisplaySetService.js

```diff
@@ -30,6 +30,7 @@ const findInstance = (instance, displaySets) => {
 
 export default class DisplaySetService {
   constructor() {
+    this._unfilteredDisplaySets = new Map();
```

Comment:
I would put this at the end of constructor personally (instead of first thing), but not important

Reply:
weird, me too!